app/(tabs)/(home)/index.tsx

```diff
@@ -37,6 +38,7 @@ export default function Index() {
   const [longitude, setLongitude] = useState(8.24); // Default to Wiesbaden
   const [locationServicesEnabled, setLocationServicesEnabled] = useState(false);
   useEffect(() => {
+    initializeDatabase();
```

Comment:
Es wäre sinnvoller, initializeDatabase() in app/_layout.tsx auszuführen, weil das die Rootkomponente ist. Marco(?) hat dafür in Zeile 24 schon einen Platzhalter eingefügt.

Reply:
Ändere ich im anderen Branch 👍